docker/Dockerfile.prod

```diff
@@ -0,0 +1,26 @@
+FROM golang:1.15
+
```

Comment:
For security, let's introduce a user so we don't run our app as privileged root. I refer to `Let’s build a more secure docker image` section of the following article and it has served me well. Maybe have a look!
https://medium.com/@chemidy/create-the-smallest-and-secured-golang-docker-image-based-on-scratch-4752223b7324